tests/phpunit/tests/user/retrievePassword.php

```diff
@@ -66,12 +66,12 @@ static function() {
 	public function test_retrieve_password_should_fetch_user_by_login_if_not_found_by_email() {
```

Comment:
Let's add a `@ticket 58311` to the docblock so that folks know this was touched by multiple tickets.